packages/shared/src/hooks/post/useSmartTitle.ts

```diff
@@ -29,21 +29,15 @@ export const useSmartTitle = (post: Post): UseSmartTitle => {
   const { displayToast } = useToastNotification();
   const { user, isLoggedIn } = useAuthContext();
   const { logEvent } = useLogContext();
-  const { isPlus, showPlusSubscription } = usePlusSubscription();
+  const { isPlus } = usePlusSubscription();
   const { completeAction } = useActions();
   const { flags } = useSettingsContext();
 
   const { clickbaitShieldEnabled } = flags || {};
 
   const key = useMemo(
-    () => [
-      ...getPostByIdKey(post?.id),
-      {
-        key: 'title',
-        showPlusSubscription,
```

Comment:
@omBratteng do you think we still need to have this property for the QueryKey? Like

```
{
  key: 'title',
  showPlusSubscription: true
}
```